### PR TITLE
Misc improvements mostly Jenkins 

### PR DESCRIPTION
**Pull request by danghai**
I make PR to supersedes #52

### CODE REVIEW

**Comment by danghai** [comment]
@spbnick It is good to review now!

**Comment by danghai** [comment]
@spbnick I have updated! Let me know if you need anything change.

**Comment by spbnick** [comment]
@veruu, could you please review this again? @danghai rebased and cleaned it up. I think it would be great to get this in.

File: sktm/jenkins.py [hunk 1]
```diff
@@ -159,136 +154,139 @@ def __get_cfg_data_max(self, jobname, buildid, stepname,
...
-        return "%s/job/%s/%s" % (self.server.base_server_url(), jobname,
+    def get_result_url(self, buildid):
+        """
+        Get the URL of the web representation of the specified build.
+
+        Args:
+            buildid:    Jenkins build ID.
+
+        Return:
+            The URL of the build result.
```
Comment (veruu) [inline sktm/jenkins.py]: I can't comment on a specific commit, but the `Describe sktm.jenkins.get_result_url functions ` commit is only adding docstring to this function, while in the original commit it's also documenting `get_result` one (hence why there is an asterisk in the original commit message, which is also missing in this one).

File: sktm/__init__.py [hunk 2]
```diff
@@ -312,60 +296,59 @@ def check_patchwork(self):
...
                                 self.jk.build(
-                                    self.jobname,
                                     baserepo=self.baserepo,
                                     ref=stablecommit,
                                     baseconfig=self.cfgurl,
                                     message_id=series.message_id,
                                     subject=series.subject,
                                     emails=series.email_addr_set,
-                                    patchwork=series.get_patch_url_list(),
+                                    patchwork=url_list,
```
Comment (veruu) [inline sktm/__init__.py]: The method parameter is still named `patchwork` here (while it's changed in the `build()`, so this commit would break things)

**Comment by spbnick** [comment]
Thank you, @veruu for spotting the omissions! Approving and handing over to you now :)

**Comment by spbnick** [comment]
@danghai, do you think you could rebase this again and address @veruu's comments?

**Comment by danghai** [comment]
@spbnick @veruu I rebased and addressed comments. It is good to review now :)

**Comment by spbnick** [comment]
Thank you very much, @veruu and @danghai :smile:!

**Pull request merged**